modules/installation-osp-setting-cloud-provider-options.adoc

```diff
@@ -7,11 +7,11 @@
 
 :_content-type: PROCEDURE
 [id="installation-osp-setting-cloud-provider-options_{context}"]
-= Setting cloud provider options
+= Setting OpenStack Cloud Controller Manager options
 
-Optionally, you can edit the cloud provider configuration for your cluster. The cloud provider configuration controls how {product-title} interacts with {rh-openstack-first}.
+Optionally, you can edit the OpenStack Cloud Controller Manager (CCM) configuration for your cluster. This configuration controls how {product-title} interacts with {rh-openstack-first}.
 
-For a complete list of cloud provider configuration parameters, see link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+For a complete list of configuration parameters, see the "OpenStack Cloud Controller Manager reference guide" page in the "Installing on OpenStack" documentation.
```

Comment:
Do we not have the ability to cross-link our own docs? We documented all of this recently, right?

Reply:
@stephenfin Not within a module like this, no. 

It's a limitation that everyone hates. 👍

Reply:
And yeah, this is just to clean up some linking and language in a couple of versions of the docs that suffered a bad rebase at some point.